aleph-client/src/pallets/proxy.rs

```diff
@@ -0,0 +1,79 @@
+use subxt::utils::MultiAddress;
+
+use crate::{
+    aleph_runtime::{ProxyType, RuntimeCall},
+    api, AccountId, SignedConnectionApi, TxInfo, TxStatus,
+};
+
+/// any object that implements pallet proxy api
+#[async_trait::async_trait]
+pub trait ProxyUserApi {
+    /// API for [`proxy`](https://paritytech.github.io/polkadot-sdk/master/pallet_proxy/pallet/struct.Pallet.html#method.proxy) call.
```

Comment:
This comment is just a link to the original one. In the browser, like azero.dev, when you choose the pallet/endpoint it will show the comment. How does it work here? Which comment will appear there?
<img width="1364" alt="Screenshot 2023-11-15 at 13 19 59" src="https://github.com/Cardinal-Cryptography/aleph-node/assets/95355183/702419cd-cb8e-4f55-8d39-94e4dfa83f7e">